spec/tla/run-iterator-validation.sh

```
@@ -0,0 +1,86 @@
+#!/bin/bash
+# Run all TLA+ iterator model specs and verify expected results.
+# Each spec has: correct config (must PASS), bug config (must FAIL), canary config (must FAIL).
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)